django/db/backends/oracle/operations.py

```diff
@@ -678,9 +678,14 @@ def bulk_insert_sql(self, fields, placeholder_rows):
                     internal_type = getattr(
                         fields[i], "target_field", fields[i]
                     ).get_internal_type()
-                    placeholder = (
-                        BulkInsertMapper.types.get(internal_type, "%s") % placeholder
-                    )
+                    if not (
+                        internal_type == "DateTimeField"
+                        and placeholder == "CURRENT_TIMESTAMP"
+                    ):
```

Comment:
There might be a nicer way to do this, but I haven't looked into it particularly.

Reply:
See #16847.